Changelog.md

```diff
@@ -1,3 +1,48 @@
+# 1.2
+
+NOTE: haskell-ide-engine is in transition, the new home for it will be
+[haskell-language-server](https://github.com/haskell/haskell-language-server),
+which does not currently have feature parity with `hie`. In particular
+it does not support multi-cradles, so can only open a single component
+of a project, as configured into its `hie.yaml` file.
+
+Once that hurdle is crossed, the main emphasis will be there, and
+`hie` will eventually be deprecated.
+
+## In this version
+
+- cabal now index state 2020-03-01T07:49:28Z
+- GHC 8.8.2 is lts-15.2
+- GHC 8.6.5 is lts-14.22
+- hlint is 2.2.11
+- brittany is 0.12.1.1
```

Comment:
Maybe I am missing something, but isnt in this PR the lower haskell-src-exts version bumped as well?